<launch>
	<arg name="uav_id" default="1"/>
    <arg name="sim_mode" default="true"/>
    <arg name="flag_printf" default="true"/> 

    <!-- 启动uav_control_main -->
    <arg name="input_source" default="3"/>
	<node pkg="prometheus_uav_control" type="uav_control_main" name="uav_control_main_$(arg uav_id)" output="screen">
		<param name="uav_id" value="$(arg uav_id)" />
        <param name="sim_mode" value="$(arg sim_mode)" />
		<param name="flag_printf" value="$(arg flag_printf)" />
        <param name="control/location_source" value="$(arg input_source)"/>
		<rosparam command="load" file="$(find prometheus_uav_control)/launch/uav_control_indoor.yaml" />
	</node>
</launch>